packages/system/src/Box/stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { Box } from './Box';
+
+<Meta title="MDX/Box Component Doc" />
+
+# Box-Component
+
+<br />
+
+## Description
+
+`Box` is a mutli-purpose component and the most abstract component on top of which all other Marigold UI components are built.
+The `Box` allows us to apply styling via a dedicated prop (`css`) while respecting the rules and constraints of our design system.
+Instead of exposing the underlying tools that will create CSS, the `Box` component wraps them and exposes a more React-like API for applying styling to a component.
+
+<br />
+
+### Style components
+
+The style of components is defined by base, theme and style props. These three layers are structured hierarchically. A layer can overwrite the attributes of the previous layer.
+
+**base < theme < style props**
+
+- **base** <br />
+  The base style is the minimum of styles each component has. With the base style the component is displayed normalized in every browser.
```

Comment:
Is not really about the "minimum of styles", it's about giving some basic styling to a component. It should be used to have elements looks the same in every browser, but it could do more than that. E.g. in a `<Text>` component this can be used to the `fontfamily` is alsways used from the `theme.fontFamily.body` value.